test/test_jit.py

```diff
@@ -10294,6 +10294,19 @@ def subscript_tuple_augmented_assign(a):
         with self.assertRaisesRegex(RuntimeError, 'does not support augmented assign'):
             scripted_aug_assign = torch.jit.script(subscript_tuple_augmented_assign)
 
+    def test_multiple_assign(self):
+        def test():
+            a = b, c = d, f = (1, 1)
```

Comment:
I do not see any tests for lvalues that are not variables. Do they work? That was the primary reason for omitting this before.

Reply:
there is also pattern matching assignment in the test. what else were you thinking ?